_posts/2015-07-19-pre-ks-announcement.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: We're launching a Kickstarter SOON!
+title: Kickstarter Incoming!
```

Comment:
I'm not sure this is significantly more indicative of the post content. 

Maybe something boring but summarizing like "Meta Mesh to launch crowdfunding of mesh networking equipment".